plugin/helpers_single_node.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package plugin
+
+import "fmt"
+
+// RunOnSingleNode is a wrapper function which should guarantee that only one plugin instance can run a function associated with the given id at a time
```

Comment:
We don't need to duplicate the document here. This can just be:
```suggestion
// RunOnSingleNode implements Helper.RunOnSingleNode
```